src/index.css

```diff
@@ -3,9 +3,12 @@
 @tailwind utilities;
 
 @import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
 
 body {
-  font-family: 'Lato', sans-serif;
+  font-family: "Montserrat", sans-serif;
+  background: #f5f6fa;
 }
```

Comment:
- [ ] Please remove the grey area around the body element to match the zeplin design.

![image](https://github.com/DjibrilM/book-store/assets/91486791/e2c2f662-8f49-4616-9182-a048681cca5b)